### DISCUSSION TITLE
Error after convergence of first transient step

### DISCUSSION THREAD

**Discussion by AngeloCass** [idaholab/moose#17686] [Q&A General] [answered]
Hi,

I am currently working with a graduate student who is using MOOSE to build and run some model simulations. He is currently unavailable to answer questions about an error that I am getting, and I have a limited knowledge on the framework. The error occurs once the first transient step converges and the second step begins. 

The error message (last 2 lines) with some surrounding terminal output is as follows:

        120 Nonlinear |R| = 1.029252e-06
              0 Linear |R| = 1.029252e-06
              1 Linear |R| = 3.706443e-09
              2 Linear |R| = 1.795191e-11
              3 Linear |R| = 4.848944e-14
          Linear solve converged due to CONVERGED_RTOL iterations 3
        121 Nonlinear |R| = 8.804651e-08
        Nonlinear solve converged due to CONVERGED_FNORM_ABS iterations 121
         Solve Converged!
        
        Time Step 2, time = 0.2, dt = 0.1
         0 Nonlinear |R| = 3.808426e-02
         ** On entry to DGEMM parameter number  8 had an illegal value
         ** On entry to DGEMM parameter number  8 had an illegal value

If anyone knows why this may be happening, it would be a great help to me.

Thanks!

**Comment by aeslaughter** [comment]
@fdkong Can you help with this? My googling indicates that it might be coming from BLAS.

**Reply by AngeloCass** [reply]
I did some further testing and it appears that this issue is related to adaptivity. When disabling adaptivity in the input file, the simulation runs completely. Since it is desired to use the adaptivity function, I am hoping to solve this issue without completely disabling adaptivity.

**Comment by fdkong** [comment]
It might be related to a direct solver you are using. Are you using super_dist? What does the executioner block look like?

**Reply by AngeloCass** [reply]
Here's the executioner block.
[ExecutionerBlock.txt](https://github.com/idaholab/moose/files/6416729/ExecutionerBlock.txt)

**Reply by fdkong** [reply]
It looks normal to me.

**Comment by fdkong** [comment]
@hugary1995 @lindsayad 

Here is a similar case. Did you guys dig into the global AD indexing things?

**Reply by lindsayad** [reply]
No @hugary1995 suggested he might be able to make a MOOSE input at which point I could dig into it

**Reply by lindsayad** [reply]
I just need an input file I can run to reproduce

**Reply by hugary1995** [reply]
Sorry I got distracted by other stuff. I am working on this right now.

**Reply by hugary1995** [reply]
I created the issue #17755 

@AngeloCass See the issue above. I have a hunch that your error was due to the same issue. If you could also simplify your input file with MOOSE-only objects and create an issue, it might help @lindsayad fixing the bug.

**Comment by hugary1995** [comment, marked as answer]
@lindsayad has fixed the issue. If you grab the newest version of moose, your issue should be fixed.

**Reply by AngeloCass** [reply]
Thank you very much! I'll pull down the newest version.
